set.mm

```diff
@@ -425735,15 +425807,11 @@ by two edges to a closed walk v(0) ... v(n)=v=v(0) in exactly one way.
       IUUBAXOWJWKWLXPXTWMWNWOXFYDXQYFYAUKXFYCXPSXDXCYJYCXPTXEYKYLABDEFGHIJKLMNO
       WPURUSXFYEXTSXCXEYEXTTXBABCEGIJKPWRWSUSWTXA $.
 
-    $d C w $.  $d F w $.
-    $( Statement 12 in [Huneke] p. 2:  "Thus f(n) = (k - 1)f(n - 2) + k^(n-2)."
-       - the number of the closed walks v(0) ... v(n-2) v(n-1) v(n) is the sum
-       of the number of the closed walks v(0) ... v(n-2) v(n-1) v(n) with
-       v(n-2) = v(n) (see ~ numclwwlk1 ) and with v(n-2) =/= v(n) ( see
-       ~ numclwwlk2 ): f(n) = kf(n-2) + k^(n-2) - f(n-2) = (k - 1)f(n - 2) +
-       k^(n-2).  (Contributed by Alexander van der Vekens, 26-Aug-2018.)
-       (Revised by AV, 1-Jun-2021.) $)
-    numclwwlk3 $p |- ( ( ( G RegUSGraph K /\ G e. FriendGraph )
+    $d F w $.
+    $( Obsolete version of ~ numclwwlk3 as of 21-Jan-2022.  (Contributed by
+       Alexander van der Vekens, 26-Aug-2018.)  (Revised by AV, 1-Jun-2021.)
+       (Proof modification is discouraged.)  (New usage is discouraged.) $)
+    numclwwlk3OLD $p |- ( ( ( G RegUSGraph K /\ G e. FriendGraph )
```

Comment:
generally, one puts the OLD after the new statement, but do not bother if this requries additional changes

Reply:
To move the OLD theorem is not possible in this case, because it is inside a block. In about a year, this will be removed anyway...